lighthouse-core/gather/driver.js

```diff
@@ -113,8 +113,9 @@ class Driver {
       // Not mandatory but not used much
       'blink.console',
 
-      // Most the events we need come in on these two
+      // Most the events we need come in on these three
       'devtools.timeline',
+      'devtools.timeline.async',
```

Comment:
what's this provide?

Reply:
I don't think I actually needed the events it adds for any changes here, but it provides the [flow events](https://cs.chromium.org/search/?q=devtools.timeline.async+-f:out+lang:cc&type=cs) to track task root initiators. We'll need it eventually to tackle #8526, but I'll revert for now.